src/Sulu/Component/SmartContent/Tests/Unit/Orm/DataProviderRepositoryTraitTest.php

```diff
@@ -161,10 +155,7 @@ public function appendJoins(QueryBuilder $queryBuilder, $alias, $locale): void
 
     public function testFindByFiltersIdsWithDatasourceWithoutIncludeSubFolders(): void
     {
-        $findByFiltersIdsReflection = new \ReflectionMethod(
-            \get_class($this->dataProviderRepositoryTrait),
-            'findByFiltersIds'
-        );
+        $findByFiltersIdsReflection = (new \ReflectionClass(\get_class($this->dataProviderRepositoryTrait)))->getMethod('findByFiltersIds');
```

Comment:
This change in PHP 8.3 was reverted and so is not longer required: https://github.com/php/php-src/issues/9470